### ISSUE TITLE
Description update issue should be removed from README.md

### ISSUE THREAD

**Issue by DamaniN** [terraform-aws-modules/terraform-aws-security-group#144]
Since #52 removes the limitation of updating the security group description the following note should be removed from the README.md:

```
Known issues

Due to an issue #1920 in AWS provider, updates to the description of security group rules are
ignored by this module. If you need to update description after the security group has been
created you need to recreate security group rule.```

**Comment by SolidSly** [comment]
Cf #150

**Comment by antonbabenko** [comment]
Fixed in [v3.3.0](https://github.com/terraform-aws-modules/terraform-aws-security-group/releases/v3.3.0).
